src/main/webapp/nav.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
```

Comment:
Since this is being included directly into the other pages, I think you'll want to remove all tags except `<nav>` and its contents (i.e. this is not a stand-alone page itself, so it doesn't need to be "valid" HTML on its own).

Otherwise the resulting page will have multiple nested `<html>`, `<body>`, etc. (unless the CSI library somehow removes this?).